src/parser/type_checking.c

```diff
@@ -1,5 +1,6 @@
 /*
- * Copyright (C) 2008 Search Solution Corporation. All rights reserved by Search Solution.
+ * Copyright (C) 2008 Search Solution Corporation
+ * Copyright (C) 2016 CUBRID Corporation
```

Comment:
This file isn't modified. we don't need to update license

Reply:
ok. I will do that next time